Makefile

```diff
@@ -9,6 +9,8 @@ endif
 COMMA=,
 PACKAGE_NAME=async-profiler-$(PROFILER_VERSION)-$(OS_TAG)-$(ARCH_TAG)
 PACKAGE_DIR=/tmp/$(PACKAGE_NAME)
+SYM_PACKAGE_NAME=$(PACKAGE_NAME)-sym
+SYM_PACKAGE_DIR=$(PACKAGE_DIR)-sym
```

Comment:
Is it a common naming convention? I saw `-debug` or `-debuginfo`, but not `-sym`.

Reply:
Moving to `-debug`.